packages/client/hmi-client/src/components/workflow/ops/calibrate-ensemble-ciemss/calibrate-ensemble-util.ts

```diff
@@ -97,6 +99,21 @@ export function getSelectedOutputEnsembleMapping(
 			datasetMapping: wfOutputState?.timestampColName ?? '',
 			modelConfigurationMappings: {}
 		});
+
+	// For every State Variable that has not been mapped in the ensembleMapping
+	// We will fill in here so the user can still see these if they want
+	const allStates = Object.keys(stateToModelConfigMap);
+	allStates.forEach((state) => {
```

Comment:
Why the need to create a variables?
```suggestion
	Object.keys(stateToModelConfigMap).forEach((state) => {
```

Reply:
No explicit need
I have been told in the past it makes it easier to read/follow to break chunks like that up so I did